src/main/java/com/exasol/adapter/dialects/snowflake/SnowflakeSqlDialect.java

```diff
@@ -1,55 +1,57 @@
-package com.exasol.adapter.dialects.athena;
+package com.exasol.adapter.dialects.snowflake;
 
-import static com.exasol.adapter.AdapterProperties.*;
+import static com.exasol.adapter.AdapterProperties.CATALOG_NAME_PROPERTY;
+import static com.exasol.adapter.AdapterProperties.SCHEMA_NAME_PROPERTY;
 import static com.exasol.adapter.capabilities.AggregateFunctionCapability.*;
 import static com.exasol.adapter.capabilities.LiteralCapability.*;
 import static com.exasol.adapter.capabilities.MainCapability.*;
 import static com.exasol.adapter.capabilities.PredicateCapability.*;
 import static com.exasol.adapter.capabilities.ScalarFunctionCapability.*;
+import static com.exasol.adapter.dialects.SqlDialect.StructureElementSupport.MULTIPLE;
+import static com.exasol.adapter.dialects.SqlDialect.StructureElementSupport.SINGLE;
 
 import java.sql.SQLException;
-import java.util.*;
+import java.util.Set;
 
 import com.exasol.adapter.AdapterProperties;
 import com.exasol.adapter.capabilities.Capabilities;
 import com.exasol.adapter.dialects.*;
 import com.exasol.adapter.jdbc.*;
 
 /**
- * This class implements the SQL dialect of Amazon's AWS Athena.
+ * This class implements the SQL dialect of Snowflake.
  *
- * @see <a href="https://aws.amazon.com/athena/">AWS Athena</a>
+ * @see <a href="https://snowflake.com/">SNOWFLAKE</a>
  */
-public class AthenaSqlDialect extends AbstractSqlDialect {
-    static final String NAME = "ATHENA";
+public class SnowflakeSqlDialect extends AbstractSqlDialect {
+    static final String NAME = "SNOWFLAKE";
     private static final Capabilities CAPABILITIES = createCapabilityList();
 
     private static Capabilities createCapabilityList() {
-        return Capabilities //
-                .builder() //
+        return Capabilities.builder()
                 .addMain(SELECTLIST_PROJECTION, SELECTLIST_EXPRESSIONS, FILTER_EXPRESSIONS, AGGREGATE_SINGLE_GROUP,
                         AGGREGATE_GROUP_BY_COLUMN, AGGREGATE_GROUP_BY_EXPRESSION, AGGREGATE_GROUP_BY_TUPLE,
-                        AGGREGATE_HAVING, ORDER_BY_COLUMN, ORDER_BY_EXPRESSION, LIMIT) //
-                .addLiteral(NULL, BOOL, DATE, TIMESTAMP, TIMESTAMP_UTC, DOUBLE, EXACTNUMERIC, STRING, INTERVAL) //
+                        AGGREGATE_HAVING, ORDER_BY_COLUMN, ORDER_BY_EXPRESSION, LIMIT)
+                .addLiteral(NULL, BOOL, DATE, TIMESTAMP, TIMESTAMP_UTC, DOUBLE, EXACTNUMERIC, STRING, INTERVAL)
```

Comment:
Add the missing comment markers at the end to prevent the auto-formatter from removing the intentional line breaks.